src/Tools/Common/Commands/ProcessStatus.cs

```diff
@@ -37,36 +118,55 @@ public static void PrintProcessStatus(IConsole console)
                     .ThenBy(process => process.Id);
 
                 var currentPid = Process.GetCurrentProcess().Id;
-
+                List<(int processId, string processName, string fileName, string cmdLineArgs)> printInfo = new ();
                 foreach (var process in processes)
                 {
                     if (process.Id == currentPid)
                     {
                         continue;
                     }
-
                     try
                     {
-                        sb.Append($"{process.Id, 10} {process.ProcessName, -10} {process.MainModule.FileName}\n");
+                        String cmdLineArgs = GetArgs(process);
+                        cmdLineArgs = cmdLineArgs == process.MainModule?.FileName ? string.Empty : cmdLineArgs;
+                        string fileName = process.MainModule?.FileName ?? string.Empty;
+                        string[] cmdList = cmdLineArgs.Split(" ");
+                        foreach(string str in cmdList)
+                        {
+                            if (str == string.Empty)
+                            {
+                                break;
+                            }
+                            //Assume the first string to contain the directory separation character is the filepath
```

Comment:
Could you add an if statement here that checks to make sure the first argument has a path separator? And if it doesn't then treat it like an argument instead of the name of the process

Reply:
When I use [Path.PathSeparator](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.pathseparator?view=net-6.0), it is a colon, I am not sure why. But I can check what OS they are and assign roles that way.

Reply:
You probably want to use `Path.DirectorySeparatorChar`; [Path.PathSeparator](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.pathseparator?view=net-6.0) is to separate paths when they are in a list (like the PATH variable).